ckan/templates/package/base.html

```diff
@@ -21,6 +21,6 @@
     <li{{ self.breadcrumb_content_selected() }}>{% link_for dataset|truncate(30), named_route=pkg.type ~ '.read', id=pkg.id if is_activity_archive else pkg.name %}</li>
   {% else %}
     <li>{% link_for _(dataset_type.title()), named_route=dataset_type ~ '.search' %}</li>
-    <li class="active"><a href="">{{ _('Create Dataset') }}</a></li>
+    <li class="active"><a href="">{{ _('Create ' ~ dataset_type.title()) }}</a></li>
```

Comment:
this kind of string can't be picked up by `extract_messages` but I guess we've done the same in the earlier PR.

Reply:
See my suggestions in #5337 I think we should do the same here